tests/modules/gappa/examineheattree/test.yml

```diff
@@ -0,0 +1,19 @@
+- name: "gappa examineheattree"
+  command: nextflow run ./tests/modules/gappa/examineheattree -entry test_gappa_examineheattree -c ./tests/config/nextflow.config -c ./tests/modules/gappa/examineheattree/nextflow.config
+  tags:
+    - "gappa"
+    - "gappa/examineheattree"
+  files:
+    - path: output/gappa/test.tree.newick
+      md5sum: f055e11c9b44b6d3afb502aba99d9579
+    - path: output/gappa/test.tree.nexus
+      md5sum: e1cfdcb63f3a751677d0e78fb6b9230a
+    - path: output/gappa/test.tree.phyloxml
+      md5sum: ea35b07e4fd8d174fd6b2f204629cc4e
+    - path: output/gappa/test.tree.svg
+    - path: output/gappa/test.colours.txt
+      md5sum: 7704cd341c8693981253ba6d740c1061
+    - path: output/gappa/test.log
+    - path: output/gappa/versions.yml
+      contains:
+        - "0.8.0"
```

Comment:
I am a little lost why the versions.yml should vary in content (md5sum)? If it only contains the version string as tested for here it shouldnt vary?!

Reply:
I agree. I got variation a while ago for this container and have since just repeated that.